### PR TITLE
Update Footer

### PR DESCRIPTION
**Pull request by ayushkoli772**
New footer will look like this- 

image

@adpi2 Please review. Thank you

### CODE REVIEW

File: modules/template/src/main/twirl/scaladex/view/main.scala.html [hunk 1]
```diff
@@ -107,32 +107,54 @@
...
+                      <li><a href="https://github.com/scalacenter/scaladex?tab=BSD-3-Clause-1-ov-file">License</a></li>
+                  </ul>
+              </div>
+              <div class="col-md-3">
+                  <ul class="social">
+                      <li><h5>Social</h5></li>
+                      <li><a href="https://twitter.com/scala_index"><i class="fa-brands fa-twitter fa-lg"></i>Twitter</a></li>
+                      <li><a href="https://github.com/scalacenter/scaladex"><i class="fa-brands fa-github fa-lg"></i>Github</a></li>
+                  </ul>
+              </div>
```
Comment (adpi2) [inline modules/template/src/main/twirl/scaladex/view/main.scala.html:141]: `Community` and `Social` is a bit redundant. The Github link can be moved to the `Community` section, as suggested above. Also the `Twitter` account is dead so we can remove the link completely.

```suggestion
```

**Review by adpi2 — COMMENTED** [review]
Thanks! I have some suggestions regarding the content, but apart from that it looks beautiful!

**Comment by ayushkoli772** [comment]
Done.

This is how it will look-

Screenshot from 2024-04-15 15-47-43

@adpi2 Please review

Thank you

**Comment by ayushkoli772** [comment]
Done. Please review

**Review by adpi2 — APPROVED** [review]

**Pull request merged**